test/unit/test-ampdoc.js

```diff
@@ -54,17 +95,17 @@ describe('AmpDocService', () => {
 
     it('should initialize as single-doc', () => {
       expect(service.isSingleDoc()).to.be.true;
-      expect(service.singleDoc_).to.exist;
-      expect(service.singleDoc_).to.be.instanceOf(AmpDocSingle);
+      expect(service.getSingleDoc()).to.exist;
+      expect(service.getSingleDoc()).to.be.instanceOf(AmpDocSingle);
     });
 
     it('should always yield the single document', () => {
       expect(() => service.getAmpDoc(null)).to.throw;
-      expect(service.getSingleDoc()).to.equal(service.singleDoc_);
-      expect(service.getAmpDoc(document)).to.equal(service.singleDoc_);
+      expect(service.getSingleDoc()).to.equal(service.getSingleDoc());
```

Comment:
seems can be removed? or are you testing it's the same instance? `to.be.same(xxx)`

Reply:
You're right. A meaningless test.